helm_deploy/hmpps-prisoner-to-nomis-update/templates/activities-purge-dlq-cronjob.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-activities-purge-dlq
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "9-59/10 * * * *"
```

Comment:
The retryDLQ job runs every 10 minutes from 00 so we clear the DLQ prior to this.